test-authenticator/pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>io.curity.identityserver.plugin</groupId>
     <artifactId>identityserver.plugins.authenticators.test</artifactId>
-    <version>1.0.0</version>
+    <version>1.1.0</version>
```

Comment:
Should this be `1.1.0-SNAPSHOT` since this is a Maven project and the SDK isn't yet _final_ for the 5.4.0 release?

Reply:
We need to have a strategy for all plugins, so if you want to use SNAPSHOT for all of them until we have a final SDK release, I am ok with that.

Reply:
I was thinking released, not final final (i.e., not 6.0 but 5.4). Otherwise, people will end up using this plug-in with a dependency from Nexus instead of Maven Central (where 5.4 of the SDK will be published when it's ready)

Reply:
Changed the version to 1.1.0-SNAPSHOT and target dev branch (created it from master).